frontend/fragments/try-footer.html

```diff
@@ -448,6 +471,17 @@ <h2 id="like-this"> Like This? </h2>
   document.location = url;
 }
 
+function generateDynamicImage() {
+  var url = '/badge/dynamic/' + escapeField(dynamicImageMaker.type.value);
```

Comment:
I think you want to start with the `var url = document.getElementById('imgUrlPrefix').textContent` line from `makeImage()` above. Otherwise it will try to serve from `shields.io` instead of `img.shields.io` in production.

Reply:
To test this you can run `make website`. The dynamic badge won't work of course, though you should be able to confirm that it's generating URLs that begin with `https://img.shields.io/`.

Reply:
Ah did not think about that, i updated it to match the static badge function.